homeassistant/components/sky_hub/device_tracker.py

```diff
@@ -67,45 +75,11 @@ def _update_info(self):
 
         _LOGGER.info("Scanning")
 
-        data = _get_skyhub_data(self.url)
+        data = await self._hub.async_get_skyhub_data()
 
         if not data:
-            _LOGGER.warning("Error scanning devices")
             return False
 
         self.last_results = data
 
         return True
```

Comment:
We can remove the return statement.

Reply:
I've not changed that line, have the code requirements changed?

Reply:
Removed